### DISCUSSION TITLE
External Access

### DISCUSSION THREAD

**Discussion by Dorin-IDE** [stashapp/stash#3886] [General]

**Comment by ALonelyJuicebox** [comment]
Stash isn't really designed for external use in the way you're intending and using it for that purpose can expose both the computer hosting Stash as well as and your network to security risks that you may not be prepared to defend against. 

I recommend instead gating access to Stash from behind something like Tailscale where your friends would need to auth and then be granted access to a VPN where Stash can be accessed from. Be aware that solutions like Tailscale/zerotier/etc may have usage restrictions on media streaming.

Give this a read 👍🏿
https://docs.stashapp.cc/networking/authentication-required-when-accessing-stash-from-the-internet/
